db/cache/charts.go

```diff
@@ -68,7 +69,7 @@ const (
 )
 
 // cacheVersion helps detect when cache data stored has its structure changed.
-var cacheVersion = semver.NewSemver(1, 0, 0)
+var cacheVersion = semver.NewSemver(1, 1, 0)
```

Comment:
Does any difference in the semantic version result in a complete recompute of all charts data?  Ideally a minor version bump (1.0.0 -> 1.1.0) would allow loading a cache dump with the previous minor version (1.0.0) and then completely building just the added chart data and updating everything else.  If that is not current behavior, it seems like we should simply bump the major version.  If that behavior is not planned, then we should just be using an integer version.  Can you clarify, @dmigwi?  Also, the docs for `cacheVersion` should explain how the version components are used and what actions are taken when the version changes.

Reply:
Yes. Recomputing of all charts data was the intended effect effect in case of a version change.

I will add the needed documentation.

Reply:
In that case, what is the meaning of each major/minor/patch component of the semver if any change requires recomputing all charts data?  If there are no plans to ever have a partial recompute with minor updates, or patching/checks with a patch update, then I suggest we just bump the major version, and change all logging to report the version as a single integer (e.g. cache dump file v2).